src/main/resources/listLookup.json

```diff
@@ -0,0 +1,86 @@
+{
+  "SJP_PUBLIC_LIST": {
+    "friendlyName": "Single Justice Procedure Public List",
+    "shortenedFriendlyName": "SJP Public List",
+    "url": "sjp-public-list",
+    "jurisdictions": ["Single Justice Procedure"],
+    "restrictedProvenances":  []
```

Comment:
If this field used anywhere? The field value for all list types are empty except the SJP press list. Are we supposed to  include the actual allowed provenance like in the back end?